libs/dh/core/shell/src/lib/dh-core-shell.component.spec.ts

```diff
@@ -18,17 +18,17 @@ import { By } from '@angular/platform-browser';
 import { WattShellComponent } from '@energinet-datahub/watt';
 import { render, RenderResult } from '@testing-library/angular';
 
-import { ShellComponent } from './shell.component';
-import { ShellModule } from './shell.module';
+import { DhCoreShellComponent } from './dh-core-shell.component';
+import { DhCoreShellModule } from './dh-core-shell.module';
 
-describe(ShellComponent.name, () => {
+describe(DhCoreShellComponent.name, () => {
   beforeEach(async () => {
-    view = await render(ShellComponent, {
-      imports: [ShellModule],
+    view = await render(DhCoreShellComponent, {
+      imports: [DhCoreShellModule],
     });
   });
```

Comment:
suggestion: import SCAM instead of mixed module
```suggestion
import { DhCoreShellComponent, DhCoreShellComponentScam } from './dh-core-shell.component';

describe(DhCoreShellComponent.name, () => {
  beforeEach(async () => {
    view = await render(DhCoreShellComponent, {
      imports: [DhCoreShellComponentScam],
    });
  });
```